web/src/components/Upload/UploadForm/UploadForm.tsx

```diff
@@ -21,135 +25,73 @@ interface UploadFormProps {
-          className="rw-label"
-          errorClassName="rw-label rw-label-error"
-        >
-          Agency id
-        </Label>
-
-        <NumberField
-          name="agencyId"
-          defaultValue={props.upload?.agencyId}
-          className="rw-input"
-          errorClassName="rw-input rw-input-error"
-          validation={{ required: true }}
-        />
-
-        <FieldError name="agencyId" className="rw-field-error" />
-
-        <Label
-          name="organizationId"
-          className="rw-label"
-          errorClassName="rw-label rw-label-error"
-        >
-          Organization id
-        </Label>
-
-        <NumberField
-          name="organizationId"
-          defaultValue={props.upload?.organizationId}
-          className="rw-input"
-          errorClassName="rw-input rw-input-error"
-          validation={{ required: true }}
-        />
-
-        <FieldError name="organizationId" className="rw-field-error" />
-
-        <Label
-          name="reportingPeriodId"
-          className="rw-label"
-          errorClassName="rw-label rw-label-error"
-        >
-          Reporting period id
-        </Label>
-
-        <NumberField
-          name="reportingPeriodId"
-          defaultValue={props.upload?.reportingPeriodId}
-          className="rw-input"
-          errorClassName="rw-input rw-input-error"
-          validation={{ required: true }}
-        />
-
-        <FieldError name="reportingPeriodId" className="rw-field-error" />
-
-        <Label
-          name="expenditureCategoryId"
-          className="rw-label"
-          errorClassName="rw-label rw-label-error"
-        >
-          Expenditure category id
-        </Label>
-
-        <NumberField
-          name="expenditureCategoryId"
-          defaultValue={props.upload?.expenditureCategoryId}
-          className="rw-input"
-          errorClassName="rw-input rw-input-error"
-          validation={{ required: true }}
-        />
-
-        <FieldError name="expenditureCategoryId" className="rw-field-error" />
+  const onReset = () => {
+    console.log('resetting form...')
+    formMethods.reset()
+  }
 
-        <div className="rw-button-group">
-          <Submit disabled={props.loading} className="rw-button rw-button-blue">
-            Save
-          </Submit>
-        </div>
-      </Form>
-    </div>
+  return (
+    <Form onSubmit={onSubmit}>
+      <FormError
+        error={props.error}
+        wrapperClassName="rw-form-error-wrapper"
+        titleClassName="rw-form-error-title"
+        listClassName="rw-form-error-list"
+      />
+      {/* <HiddenField name="uploadedBy">1</HiddenField> */}
+      <Label
+        name="reportingPeriodId"
+        className="rw-label"
+        errorClassName="rw-label rw-label-error"
+      >
+        Reporting Period
+      </Label>
+      <SelectField name="reportingPeriodId">
+        <option value={1}>23Q3</option>
+        <option value={2}>23Q4</option>
+        <option value={3}>24Q1</option>
```

Comment:
We're going to need a variant of an Organization Cell that takes an organizationId and returns the agency and reporting period options to use here.